src/components/Mission.js

```diff
@@ -1,58 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-// import './missions.css';
 import { useDispatch } from 'react-redux';
+// eslint-disable-next-line no-unused-vars
 import { asyncJoinMission } from '../redux/missions/missions';
 
 const Mission = (props) => {
+  // eslint-disable-next-line no-unused-vars
   const dispatch = useDispatch();
   const { mission } = props;
   const {
+    // eslint-disable-next-line no-unused-vars
     id, name, description,
   } = mission;
 
-  const handleRemove = () => {
-    dispatch(asyncJoinMission(id));
-  };
+  // const handleRemove = () => {
+  //   dispatch(asyncJoinMission(id));
+  // };
 
   // const progress = Math.floor(Math.random() * 100);
   // const chapter = Math.floor(Math.random() * 20);
```

Comment:
kindly remove code that you are not making use of, it is unprofessional to leave unused code

Reply:
ok  !
Thanks !